api/networks/bounties.js

```diff
@@ -11,7 +11,7 @@ const getGitcoin = async () => {
     return null
   }
 
-  const total = await response.json()
+  const total = await response.data
```

Comment:
just super small thing, this `await` becomes redundant with `axios` as it parses and returns JSON internally so `response.data` is simply "just there" and is not behind some promise as the `.json()` method with `fetch`

Reply:
ok, I'll remove the await